packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/FlatTreeNode.ts

```diff
@@ -13,14 +13,21 @@ interface PlaceholderNode {
 }
 
 /** @alpha */
-export type FlatNode<TNode extends PresentationTreeNode = PresentationTreeNode> = {
+export type FlatNode = {
   level: number;
   levelSize: number;
   posInLevel: number;
-} & TNode;
+} & PresentationHierarchyNode;
 
 /** @alpha */
-export type FlatTreeNode<TNode extends PresentationTreeNode = PresentationTreeNode> = FlatNode<TNode> | PlaceholderNode;
+export interface ErrorType {
+  errorNode: PresentationInfoNode;
+  parentNode?: PresentationHierarchyNode;
+  expandToNode: (expandNode: (nodeId: string) => void) => void;
```

Comment:
Whay do you think about these renames?

```suggestion
export interface ErrorNode {
  error: PresentationInfoNode;
  parent?: PresentationHierarchyNode;
  expandTo: (expand: (nodeId: string) => void) => void;
```

Reply:
Adjusted to similar, `expandTo` in my mind should continue to take `expandNode` as it is the name of the actual expand function we have in `TreeRenderer`